docs/Training-PPO.md

```diff
@@ -99,7 +111,23 @@ the action is a very complex interaction between the observation variables, this
 
 Typical Range: `32` - `512`
 
-### Training Statistics
+### (Optional) Recurrent Neural Network Hyperparameters
+
+The below hyperparameters are only used when `use_recurrent` is set to true.
+
+#### Sequence Length
+
+`sequence_length` corresponds to the length of the sequences of experience passed through the network during training. This should be long enough to capture whatever information your agent might need to remember over time. For example, if your agent needs to remember the velocity of objects, then this can be a small value. If your agent needs to remember a piece of information given only once at the beginning of an episode, then this should be a larger value.
+
+Typical Range: `4` - `128`
+
+#### Memory Size
+
+`memory_size` corresponds to the size of the array of floating point numbers used to store the hidden state of the recurrent neural network. This should be a multiple of 4, and should scale with the amount of information you expect the agent to need to remember to successfully complete the task.
```

Comment:
If memory size has to be a multiple of, say "memory_size **must** be a multiple of 4..."

Also, minor stylistic point, avoid using "this" without saying what "this" refers to. So here, say "memory size" or say "this value". If you can't easily add the noun that "this" refers to, then text probably isn't as clear as you think it is (you in general, not you, Arthur specifically :^). Using "this" without its referent also makes it harder for translations (or so I've been told).